graphql/schema.graphql

```diff
@@ -87,9 +87,9 @@ type Mutation {
   """
   generateSchedule(input: GenerateScheduleInput!): Response!
   """
-  Edit schedule
+  Edit schedule and Plug and Play Compnay 3's and 4's Algorithms
   """
-  editSchedule(year: Int!): Response!
+  editSchedule(input: EditCourseInput): Response!
```

Comment:
```suggestion
  editCourse(input: EditCourseInput): Response!
```
since you can edit capacity and stuff too, I think `editCourse` probably makes more sense

Reply:
Adding some context for discussion here.
<img width="868" alt="image" src="https://user-images.githubusercontent.com/1757744/177622498-114c940b-7c4d-420d-b57e-46e4c1bc62e5.png">